contracts/Market.sol

```diff
@@ -749,21 +755,24 @@ contract Market is
     view
     returns (uint256)
   {
-    return amount + this.calculateNoriFee({amount: amount});
+    return
+      (amount * _priceMultiple) / 100 + this.calculateNoriFee({amount: amount});
   }
 
   /**
-   * @notice Calculates the quantity of carbon removals being purchased given the purchase total and the
-   * percentage of that purchase total that is due to Nori as a transaction fee.
-   * @param purchaseTotal The total amount of Nori used for a purchase.
+   * @notice Calculates the quantity of carbon removals being purchased given the purchase total, the price multiple,
+   * and the percentage of that purchase total that is due to Nori as a transaction fee.
+   * @param purchaseTotal The total amount of ERC20 tokens used for a purchase.
    * @return The amount for the certificate, excluding the transaction fee.
    */
   function calculateCertificateAmountFromPurchaseTotal(uint256 purchaseTotal)
     external
     view
     returns (uint256)
   {
-    return (purchaseTotal * 100) / (100 + _noriFeePercentage);
+    uint256 priceWithoutFee = (purchaseTotal * 100) /
+      (100 + _noriFeePercentage);
+    return (priceWithoutFee * 100) / _priceMultiple;
```

Comment:
Can we use muldiv here?

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/math/Math.sol#L55

Reply:
Also in the line above it

Reply:
Also check the slither warning below